conjure-java-core/src/main/java/com/palantir/conjure/java/types/ErrorGenerator.java

```diff
@@ -172,12 +173,38 @@ private static JavaFile generateErrorTypesForNamespace(
                 })
                 .collect(Collectors.toList());
 
+        List<MethodSpec> isRemoteExceptionDefinitions = errorTypeDefinitions.stream()
+                .map(entry -> {
+                    String typeName = CaseFormat.UPPER_CAMEL.to(
+                            CaseFormat.UPPER_UNDERSCORE, entry.getErrorName().getName());
+                    String methodName = "is" + entry.getErrorName().getName();
+
+                    return MethodSpec.methodBuilder(methodName)
+                            .addModifiers(Modifier.PUBLIC, Modifier.STATIC)
+                            .addParameter(RemoteException.class, REMOTE_EXCEPTION_VAR)
+                            .returns(TypeName.BOOLEAN)
+                            .addStatement(Expressions.requireNonNull(
+                                    REMOTE_EXCEPTION_VAR, "remote exception must not be null"))
+                            .addStatement(
+                                    "return $N.name().equals($N.getError().errorName())",
+                                    typeName,
+                                    REMOTE_EXCEPTION_VAR)
+                            .addJavadoc(
+                                    "Returns true if the {@link $T} is named $L:$L",
```

Comment:
Slight preference for `code` tags to distinguish the name (super duper minor though)
```suggestion
                                    "Returns true if the {@link $T} is named {@code $L:$L}",
```